k8s/openebs-operator.yaml

```diff
@@ -74,7 +74,7 @@ spec:
         - name: DEFAULT_REPLICA_IMAGE
           value: "openebs/jiva:0.4.0"
         - name: DEFAULT_REPLICA_COUNT
-          value: "2"
+          value: "1"
```

Comment:
@Hrishike -- Can we set 2 itself as default.. 

An application that requires only one replica, set "replica: 1" in the volumes storage class. Example is openebs-standalone class in https://github.com/openebs/openebs/blob/master/k8s/openebs-storageclasses.yaml

Even though this depends on the resolution of : https://github.com/openebs/openebs/issues/645

Other than this.. the changes are good to be merged!

Reply:
@kmova sure. sounds apt. Done. Changed the default replicas to 2.